www/app/main.ts

```diff
@@ -5,7 +5,7 @@ import * as L from "leaflet";
 interface WorkerContainer { worker: Worker, activeJobs: Array<string>, ready: boolean }
 let workers: Array<WorkerContainer> = [];
 const initNumWorkers = Math.min(navigator.hardwareConcurrency || 4, 60);
-let maxIterations = 200, isSmoothed = true, numWorkers = initNumWorkers;
+let maxIterations = 200, exponent = 5, isSmoothed = true, numWorkers = initNumWorkers;
```

Comment:
It should probably default to 2 which is the exponent of the original Mandelbrot set